src/components/MetaData/index.test.tsx

```diff
@@ -140,6 +143,7 @@ const wrapperGenerator = (
       arcSite={arcSite}
       websiteDomain={websiteDomain}
       facebookAdmins={facebookAdmins}
+      fallbackImage={localFallback ? fallbackImageLocal : fallbackImageRemote}
```

Comment:
I'm kind of confused by this test. how does <MetaData /> know whether an image is a local image or remote?

Reply:
by the parameter `localFallback`, the test will send what to test